### PR TITLE
Issue template change: add anti-piracy warning

### PR DESCRIPTION
**Pull request by noproblemx2**
### Description of Changes
Added an anti-piracy warning to issue templates.

### Rationale behind Changes
Prevent invalid issues.

### Suggested Testing Steps
I've tested that the issue templates function correctly in my fork. There might not be any further testing required.

### CODE REVIEW

**Comment by stenzek** [comment]
-1, saying "don't include the things which show piracy" is just encouraging people to cover it up.

**Comment by noproblemx2** [comment]
> some people do have more than one BIOS file

I was thinking of instances like this https://github.com/PCSX2/pcsx2/issues/9920#issuecomment-1710650162

**Comment by stenzek** [comment]
And your point is? We don't want to help people hide piracy.

Granted most of those types of users don't read any of the notes at the beginning of the template anyway, but we don't want to get rid of one of the easy ways of filtering them out.

**Comment by noproblemx2** [comment]
I am wondering if the policy on how many BIOS files are allowed should be stated up front.

**Comment by refractionpcsx2** [comment]
We don't have a policy on that.

**Comment by stenzek** [comment]
The policy is to not pirate.

If you don't want to remove the part of the message which encourages users to hide their pirate actions, then sorry, but we'll have to close this.

**Comment by stenzek** [comment]
Okay, this is getting silly. None of the messages you have added are what we are asking for, and it feels like you're not listening.

Please change it to what refraction suggested above, otherwise we are done here.

**Comment by noproblemx2** [comment]
Done. However I am concerned this does not address the problem of genuine users who unknowingly have the incorrect amount of BIOS files being (rightly) flagged down for piracy.

**Comment by refractionpcsx2** [comment]
That's not an issue which needs solving, your example was blatant piracy.  If it looks weird but not blatant, but looks suspicious, then we will ask.

**Comment by refractionpcsx2** [comment]
Anyway, it's fine now. thanks

File: .github/ISSUE_TEMPLATE/emu_bug_report.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ body:
         If you are unsure, start with [discord](https://discord.com/invite/TCz3t9k) or the [forums](https://forums.pcsx2.net/index.php).
 
         Please make an effort to make sure your issue isn't already reported.
+
```
Comment (refractionpcsx2) [inline .github/ISSUE_TEMPLATE/emu_bug_report.yaml:20]: Can you remove the whitespace please

File: .github/ISSUE_TEMPLATE/app_bug_report.yaml [hunk 2]
```diff
@@ -17,6 +17,8 @@ body:
         If you are unsure, start with [discord](https://discord.com/invite/TCz3t9k) or the [forums](https://forums.pcsx2.net/index.php).
 
         Please make an effort to make sure your issue isn't already reported.
+
```
Comment (refractionpcsx2) [inline .github/ISSUE_TEMPLATE/app_bug_report.yaml:20]: Can you remove the whitespace please

**Comment by noproblemx2** [comment]
Thank you, done.

> That's not an issue which needs solving, your example was blatant piracy. If it looks weird but not blatant, but looks suspicious, then we will ask.

What makes that particular example piracy? I don't understand, sorry.

**Comment by refractionpcsx2** [comment]
> Thank you, done.
> 
> > That's not an issue which needs solving, your example was blatant piracy. If it looks weird but not blatant, but looks suspicious, then we will ask.
> 
> What makes that particular example piracy? I don't understand, sorry.

The chance of somebody owning those BIOS files from several different regions is very unlikely, and a Japan 1.0 console is very rare to get hold of. It looks very like a common "bios pack" people download from the internet that we've seen many times.

If they feel we've unjustly closed it, then they are free to prove ownership on the consoles and we'll be happy to reinstate them

**Comment by noproblemx2** [comment]
> The chance of somebody owning those BIOS files from several different regions is very unlikely, and a Japan 1.0 console is very rare to get hold of. It looks very like a common "bios pack" people download from the internet that we've seen many times.
> 
> If they feel we've unjustly closed it, then they are free to prove ownership on the consoles and we'll be happy to reinstate them

I understand. Then should it not be stated in the issue template that users should prove ownership of software? Otherwise users will just hide piracy, e.g. by removing BIOSs from log files.

**Comment by refractionpcsx2** [comment]
There's absolutely no need to mention it. If somebody is innocent they will question our decision. There is no need for people to prove ownership on every issue.

**Comment by refractionpcsx2** [comment]
Anyway, thanks for your contribution

**Comment by noproblemx2** [comment]
Thanks

**Pull request merged**